.travis.yml

```diff
@@ -46,7 +46,13 @@ install:
     fi
   - |
     if [ "$LINT" = "1" ] || [ "$PHP" = "1" ] || [ "$DEPLOY_PLUGIN" = "1" ]; then
-      composer install
+      composer install -n --ignore-platform-reqs
+    fi
+  - |
+    if [ $(phpenv version-name) = "5.6" ] || [ $(phpenv version-name) = "7.0" ]; then
```

Comment:
So do run in on PHP 5.6 or 7?

Reply:
This part here means: do not install / do remove phpstan on PHP <  7.1.

It only runs on 7.4 with the lint job.

But as per my other comment I might change this so it becomes less complicated.